extra-files/metavisitor/metavisitor_tool_list.yml

```diff
@@ -1,102 +1,244 @@
+- name: fastx_clipper
+  owner: devteam
+  revisions:
+  - 8192b4014977
+  tool_panel_section_id: fastx_toolkit
+  tool_panel_section_label: fastx toolkit
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+- name: fastx_reverse_complement
+  owner: devteam
+  revisions:
+  - 5bd826723d61
+  tool_panel_section_id: fastx_toolkit
+  tool_panel_section_label: fastx toolkit
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+- name: fastx_trimmer
+  owner: devteam
+  revisions:
+  - d77c9c6ecf68
+  tool_panel_section_id: fastx_toolkit
+  tool_panel_section_label: fastx toolkit
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: fetch_fasta_from_ncbi
   owner: drosofff
   revisions:
+  - 79cb7620843d
+  - 0bdc5a73c8d1
+  - befdb392fece
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: get_orfs_or_cdss
   owner: peterjc
   revisions:
   - 09a8be9247ca
+  - 705a2e2df7fb
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: khmer_normalize_by_median
   owner: iuc
   revisions:
-  - 6d8793a4923d
+  - 5531deeabd2c
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: msp_blastparser_and_hits
   owner: drosofff
+  revisions:
+  - 78c34df2dd8d
+  - 77346ecf5a28
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: msp_cap3
   owner: drosofff
+  revisions:
+  - e2ae46b16b3b
+  - ddb463fdf57e
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: msp_fasta_tabular_converter
   owner: drosofff
+  revisions:
+  - 951cb6b3979b
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: msp_oases
   owner: drosofff
+  revisions:
+  - f08d9c814ffb
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: msp_sr_bowtie
   owner: drosofff
+  revisions:
+  - c1bfa227bbb6
+  - 316124e85b8d
+  - 88cdd6118bfe
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
 - name: msp_sr_readmap_and_size_histograms
   owner: drosofff
+  revisions:
+  - dbac07fc9186
+  - ac7d8e55bb67
+  - 68f58363f1c6
+  - e4874d1ae69d
+  tool_panel_section_id: metavisitor
   tool_panel_section_label: Metavisitor
-  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  tool_shed_url: https://testtoolshed.g2.bx.psu.edu/
 - name: ncbi_blast_plus
   owner: devteam
   revisions:
+  - 623f727cdff1
+  - 577d9c12411a
   - 7f3c448e119b
-  tool_panel_section_label: Blast plus suite
+  tool_panel_section_id: ncbi_blast+
```

Comment:
if you remove all occurrences of `tool_panel_section_id` you get what you expect.